pilgrimor/exceptions.py

```diff
@@ -1,10 +1,33 @@
+from typing import Any
+
+
 class BasePilgrimorError(Exception):
     """Base error class for all Pilgrimor errors."""
 
+    error_code = 0
+    message = ""
+
+    def __init__(self, *message_args: Any, **message_kwargs: Any) -> None:
+        """Initialization.
+
+        :param message_args: Positional parameters for error message.
+        :param message_kwargs: Named parameters for error message.
+        """
+        self.message = "\n".join(  # noqa: WPS601
+            [
+                f"Exception error_code: {self.error_code}",
+                self.message.format(*message_args, **message_kwargs),
+            ],
+        )
+        super().__init__(self.message)
+
 
 class ApplyMigrationsError(BasePilgrimorError):
     """Error for unsuccessful migrations apply."""
 
+    error_code = 1
+    message = "Can't apply migrations."
```

Comment:
If you will choose second way about excpetions caughting about what I talked in privous thread, please remove message and error code.